lib/jekyll/commands/build.rb

```diff
@@ -27,6 +27,12 @@ def process(options)
           Jekyll.logger.adjust_verbosity(options)
 
           options = configuration_from_options(options)
+
+          # Enables environment setting
+          if Jekyll.env == "development"
+            options["url"] = default_url(config)
```

Comment:
you can't just move a block of code from one class to another without looking for collateral damage..
I don't think `Commands::Build` even has a `#default_url` method..